Medium/question_3.js

```diff
@@ -0,0 +1,38 @@
+// Base class
+class Item {
+  displayInfo() {
+    return "Generic item information...";
+  }
+}
+
+// Derived class - Wall
+class Wall extends Item {
+  displayInfo() {
+    return "This is a wall.";
+  }
+}
+
+// Derived class - Clock
+class Clock extends Item {
+  constructor(time) {
+    super();
+    this.time = time;
+  }
+
+  displayInfo() {
+    return `This is a clock showing the time: ${this.time}`;
+  }
+}
+
+// Function to display information about an item using polymorphism
+function displayItemInfo(item) {
+  return item.displayInfo();
+}
+
+// Creating instances of Wall and Clock
+const myWall = new Wall();
+const myClock = new Clock("12:30 PM");
```

Comment:
Prefer `let` type instead of `const`. As we may need to update class attributes or methods in main code thread.